### PR TITLE
Optimize app loading

### PR DESCRIPTION
**Pull request by sallesma**
Before sending your pull-request 💌 make sure :

- [x] The tests are up-to-date and your code is tested
- [ ] The changes you introduce are documented in the [the wiki](https://github.com/UltimateApp/UltimateApp/wiki)

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,16 +1,51 @@
...
 import Store from './src/Store/configureStore';
 
-export default class App extends React.Component {
-  render() {
-    return (
-      <Provider store={Store}>
-        <Navigation />
-      </Provider>
-    );
+enableScreens();
```
Comment (sallesma) [inline App.js:10]: This is recommended by expo : https://reactnavigation.org/docs/react-native-screens

File: App.js [hunk 2]
```diff
@@ -1,16 +1,51 @@
...
+
+const PERSISTENCE_KEY = 'NAVIGATION_STATE';
```
Comment (sallesma) [inline App.js:12]: Everything else is from this rect-navigation guide : https://reactnavigation.org/docs/state-persistence

If you think it's not useful for our users, we could also activate it only on development. It's pretty annoying to go back to the home page on reload

File: src/Navigation/Navigation.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { NavigationContainer } from '@react-navigation/native';
+import { AsyncStorage } from 'react-native';
```
Comment (arribbar) [inline src/Navigation/Navigation.js:2]: It seems you are not using `AyncStorage` in this file

Comment (sallesma) [reply src/Navigation/Navigation.js:2]: 👍

**Pull request merged**